app/views/components/Caption.scala.html

```diff
@@ -16,6 +16,5 @@
 
 @this()
 
-@(message: String, captionPrefix: String="caption.prefix")(implicit messages: Messages)
-<h2 class="govuk-caption-xl"><span class="govuk-visually-hidden">
-    @messages(captionPrefix)</span>@message</h2>
```

Comment:
@gastonracui @mywyau Was this change confirmed? I can't see any comments on the ticket about removing the `caption.prefix`

Reply:
Hmm no don't think this should of changed. My mistake will revert and run tests

Reply:
I don't think you need to revert. I can add it back in for our rebase change.